src/main/Java/jonathan/Jonathan.java

```diff
@@ -0,0 +1,257 @@
+                    taskCounter -= 1;
+                    System.out.println("     ____________________________________________________________\n" +
+                            "     Noted. I've removed this task:\n" +
+                            "       " + task + "\n" +
+                            "     Now you have " + taskCounter + " tasks in the list.\n" +
+                            "    ____________________________________________________________\n");
+
+                } else {
+                    throw new JonathanException();
+                }
+
+            } catch (ArrayIndexOutOfBoundsException e) {
+                System.out.println("    ____________________________________________________________\n" +
+                        "    The task doesn't exist, please type it correctly!\n" +
+                        "    ____________________________________________________________\n");
+
+            } catch (StringIndexOutOfBoundsException e) {
+                System.out.println("    ____________________________________________________________\n" +
+                        "    Please type with the following format:\n\n" +
+                        "    - todo <description>\n" +
+                        "    - deadline <description> /by <time>\n" +
+                        "    - event <descripton> /from <time> /to <time>\n\n" +
+                        "    note: without the angle bracket\n" +
+                        "    ____________________________________________________________\n");
+
+            } catch (JonathanException e) {
+                System.out.println("    ____________________________________________________________\n" +
+                        "    Wrong input, please type it correctly!\n" +
+                        "    ____________________________________________________________\n");
+            }
+
+        }
+    }
+
+    /*
+    Method to print the added message to the user after prompting input
+     */
+    public static void addedMessage(Task task, int taskCounter) {
+        System.out.println("    ____________________________________________________________\n" +
+                "     Got it. I've added this task:\n" +
+                "       " + task + "\n" +
+                "     Now you have " + taskCounter + " tasks in the list.\n" +
+                "    ____________________________________________________________\n");
+    }
+
+    /*
+    Method to print all the Task, including the Task's status
+     */
+    public static void lookupTask(ArrayList<Task> list, int counter) {
+        System.out.println("    ____________________________________________________________\n" +
+                "    Here are the tasks in your list:");
+        for (int i = 0; i < counter; i++) {
+            System.out.println("    " + (i+1) + ". "  + list.get(i));
+        }
+        System.out.println("    ____________________________________________________________\n");
```

Comment:
Since, you're printing the string "__________________________________" multiple times, you should store it in a variable or create a separate function to print it and use that instead. It is encouraged that no piece of code repeats itself in the files. Refer to the "DRY" Principle.